### PR TITLE
Remove commit of kvproto for every Cargo.toml

### PR DESCRIPTION
**Pull request by Little-Wallace**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:
The package manager Cargo could keep the dependency library in one commit version, so we do not need to set it in every Cargo.toml, or we must change a lot of files every time we need to update kvproto.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 
- None

### CODE REVIEW

File: src/import/sst_service.rs [hunk 1]
```diff
@@ -554,6 +554,15 @@ where
         self.threads.spawn_ok(buf_driver);
         self.threads.spawn_ok(handle_task);
     }
+
+    fn duplicate_detect(
```
Comment (BusyJay) [inline src/import/sst_service.rs:558]: I don't think it's necessary as new grpcio will provide a default implementation.

**Review by BusyJay — APPROVED** [review]
Rest LGTM

File: src/import/sst_service.rs (hunk 1 above)
Comment (Little-Wallace) [reply src/import/sst_service.rs]: I have removed it.

**Review by MyonKeminta — APPROVED** [review]

**Comment by Little-Wallace** [comment]
cc @gregwebs

**Pull request merged**